nss-ivory-trolls/BangazonAPI
# Ac setting up cors

# Description
Set up CORS policy to allow only Bangazonians to access the BagazonAPI
Updated ReadMe.md with instructions on how to set this up successfully

## Type of change



# Testing Instructions for Change Made

1. `git fetch -a`
1. `git checkout ac-setting-up-CORS
1. go to README.md and follow instructions to setup your host file
1. go to bangazon.com/api/values and ensure that all http requests are functional

# Checklist:
